Tests/SonarQube.TeamBuild.PreProcessor.Tests/Infrastructure/SampleExportXml.cs

```diff
@@ -0,0 +1,172 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="SampleExportXml.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace SonarQube.TeamBuild.PreProcessor.Tests
+{
+    /// <summary>
+    /// Examples of XML strings for testing
+    /// </summary>
+    internal static class SampleExportXml
+    {
+        #region Example of XML exported by the Java plugin
+
+        public const string RoslynExportedAdditionalFileName = "SonarLint.xml";
+        public const string RoslynExportedPackageId = "SonarLint";
+        public const string RoslynExportedPackageVersion = "1.6.0";
+        public const string RoslynExportedValidSonarLintXml = @"<?xml version=""1.0"" encoding=""utf-8""?>
```

Comment:
The exported string is large and it's used by a couple of tests so it's in a separate class that just contains constants. I could have embedded it as a resource but this was simpler.